docs/user/index.rst

```diff
@@ -1,6 +1,63 @@
 Read the Docs: Documentation Simplified
 =======================================
 
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+   :caption: Tutorials 🚀
+
+   /tutorial/index
+   /intro/getting-started-with-sphinx
+   /intro/getting-started-with-mkdocs
+   /intro/import-guide
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+   :caption: How-to guides 🪄
+
+   /guides/authors
+   /guides/administrators
+   /guides/developers
+   Troubleshooting </guides/troubleshooting/index>
+   /examples
+   /faq
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+   :caption: Explanation 💡
```

Comment:
These articles still seem oddly different in the tone of the title. It seems like eg. `Custom Domains` should be moved to features, or the title should be changed?